bin/run-ci

```diff
@@ -75,6 +77,9 @@ helm install \
   --set airflow.images.airflow.tag=$TEST_TAG \
   --set airflow.images.airflow.pullPolicy=Never \
   --set airflow.images.flower.pullPolicy=Never \
+  --set airflow.postgresql.image.registry=$POSTGRESQL_REGISTRY \
+  --set airflow.postgresql.image.repository=$POSTGRESQL_REPOSITORY \
+  --set airflow.postgresql.image.tag=$POSTGRESQL_TAG \
```

Comment:
We sure are jumping through a lot of hoops to avoid setting these values in values.yaml